### ISSUE TITLE
Inconsistent NeXus logos with poor resolution

### ISSUE THREAD

**Issue by paulmillar** [nexusformat/NIAC#145]
To the best of my knowledge, there are currently three different NeXus logos.

There are two (somewhat related) logos appearing in [NeXus for the Impatient] (https://manual.nexusformat.org/impatient/) document:
- [NeXus-logo.png](https://github.com/nexusformat/definitions/blob/main/impatient-guide/NeXus-logo.png),
- [nexuslogo.png](https://github.com/nexusformat/definitions/blob/main/impatient-guide/nexuslogo.png).

There is also a third logo, a simpler "NX" design that appears as:
- the [webpage favicon](https://www.nexusformat.org/public/apple-touch-icon-144-precomposed.png),
- the GitHub "nexusformat" [organisation icon](https://github.com/nexusformat).

For branding and consistency there are a few problems:
1. inconsistent visual identity: which logo should people used?
2. no vector graphics (e.g., EPS or SVG) version of the logo; only a (poor resolution) bitmap version is available.

Proposed resolution:
- Reduce the number of logos, ideally to just one.
- Generate an SVG version of this logo.
- Update usage to match.
- (Optionally) make a "media pack" containing the logo(s) in different formats available.
- (Optionally) write guidelines on how to use the NeXus logo.

**Comment by rayosborn** [comment]
I agree that, as a committee, we haven't been very disciplined about our branding over the years. The logo on our [documentation page](https://manual.nexusformat.org/) was the official logo when I was NIAC chair many years ago, and I have higher resolution versions, including SVG and favicon.ico files, [available here](https://anl.box.com/s/36qau4raywzkrpp62szm6undu3rbkyxl). Personally, I still quite like them, but I am biased because they were mostly my design. However, Tobias, when he was NIAC chair, disliked the pink color and I think that led to an attempt to create new logos a few years ago (I created a blue version of the old logo then). However, I don't think people thought that any of the replacements were any better so nothing changed.

My personal view is that we should change the main web page to use the Sphinx-generated pages used for the manual. I find it jarring jumping from one to the other because the designs are so unalike, and it makes the project seem unprofessional. Moving that could also be a trigger for creating a brand refresh with new logos, but none of us are professional graphic designers so I'm not sure where they will come from.

**Comment by prjemian** [comment]
Related issues: https://github.com/search?q=org%3Anexusformat+logo&type=issues

**Comment by rayosborn** [comment]
@paulmillar, the links won't disappear unless I delete them. I can probably upload them to the [NeXpy Github pages](https://github.com/nexpy), but unless the NIAC authorize it, I'm not sure I'm allowed to upload them to the [nexusformat Github](https://github.com/nexusformat). As you have pointed out, we don't yet have an official policy about logos, and I don't know how people feel about the old logo (although I guess nobody has asked for it to be removed from the Sphinx manual yet). Perhaps we can discuss it at the Telco next week. 

Since we have a code camp coming up, I would suggest we revisit changing the landing page to be Sphinx-generated.

**Comment by benajamin** [comment]
I converted the Nexus logo to SVG many years ago. I went looking and found it [here](https://github.com/nexusformat/communications/blob/9c6def935cb083fa0a0a7077fb40a06c5e6ac8f9/NeXusLogo.svg). I'm not sure why the SVG file didn't get used in the manual at the time. I think we should make more use of the SVG version.

I don't remember the version with the gradient background ever being offical, and it is advisable to avoid gradients so I think we should just delete that file.

I did try to organise a new logo around 2019-ish but I stopped the effort due to a lack of positive feedback.

**Comment by paulmillar** [comment]
Thanks for the input @benajamin!

**Comment by rayosborn** [comment]
@benajamin, by gradient background, do you mean the shadows on the letters in my version? If so, I think I agree that it's better without them.

**Comment by benajamin** [comment]
@Ray,  no, by "gradient background" I mean [this image](https://github.com/nexusformat/definitions/blob/main/impatient-guide/NeXus-logo.png) that @paulmillar originally linked to. 

The SVG files you have on ANL box requires that the viewer has the "Oxford" font installed, while the version I made uses pure paths that should always render correctly.

**Comment by phyy-nx** [comment]
@benajamin volunteered to organize this into a proposal.  Thanks!

**Comment by tacaswell** [comment]
[NeXus_Logo_Ideas_Draft3-2\[2\].pdf](https://github.com/user-attachments/files/17174246/NeXus_Logo_Ideas_Draft3-2.2.pdf)
[NeXus_Logo_Ideas_Draft4.pdf](https://github.com/user-attachments/files/17174245/NeXus_Logo_Ideas_Draft4.pdf)


A graphic designer at BNL took a shot at a redesigned logo as well.

**Comment by benajamin** [comment]
The logo of the NeXus organisation was discussed at the [NIAC2024](https://www.nexusformat.org/NIAC2024.html) meeting and SVG files representing this logo were developed. Some minor details were further discussed during the [Nov 27th telco](https://www.nexusformat.org/Telco_20241127.html). There is one official logo and a square ["favicon"](https://en.wikipedia.org/wiki/Favicon) version, with corresponding ["dark theme"](https://en.wikipedia.org/wiki/Light-on-dark_color_scheme) versions. These versions all have a consistent style and colour scheme.

The logo files have been uploaded to the [NIAC repo](https://github.com/nexusformat/NIAC/tree/master/NeXus_Logo) where they can be used in the various NeXus publications and elsewhere. Usage guidelines have been included in the README file.  I chose this location for the official, original version of the logo because it should be easy for people to find since this repo is not very busy. The SVG files are [vector graphics](https://en.wikipedia.org/wiki/Vector_graphics) and so should never look "poor resolution". EPS and PDF formats are also provided in the repo. PNG versions that are rendered with multiple resolutions are also provided in the repo for cases where bitmaps are required.

Work to update and harmonise the NeXus website and manual ([issue #1472](https://github.com/nexusformat/definitions/issues/1472)) where the new logo files will be used.

**Comment by benajamin** [comment]
I am not a lawyer and this is not legal advice:

Looking at some examples ([PSF](https://www.python.org/community/logos/), [OSI](https://opensource.org/logo-usage-guidelines)) and [advice](https://opensource.stackexchange.com/questions/2224/how-should-i-license-my-projects-logo), it seems that we don't need to provide a license for the logo files because trademark law is more applicable than copyright law in most jurisdictions. Registering a trademark costs money and the NIAC might want to discuss if it is worth doing.

**Comment by rayosborn** [comment]
@benajamin, thanks for your work on this. They look great.

**Comment by paulmillar** [comment]
Thanks for all your hard work, Ben.  I agree with @rayosborn, the logos look great!

**Comment by 7ScriptsGL** [comment]
> To the best of my knowledge, there are currently three different NeXus logos.
> 
> There are two (somewhat related) logos appearing in [NeXus for the Impatient] (https://manual.nexusformat.org/impatient/) document:
> 
> * [NeXus-logo.png](https://github.com/nexusformat/definitions/blob/main/impatient-guide/NeXus-logo.png),
> * [nexuslogo.png](https://github.com/nexusformat/definitions/blob/main/impatient-guide/nexuslogo.png).
> 
> There is also a third logo, a simpler "NX" design that appears as:
> 
> * the [webpage favicon](https://www.nexusformat.org/public/apple-touch-icon-144-precomposed.png),
> * the GitHub "nexusformat" [organisation icon](https://github.com/nexusformat).
> 
> For branding and consistency there are a few problems:
> 
> 1. inconsistent visual identity: which logo should people used?
> 2. no vector graphics (e.g., EPS or SVG) version of the logo; only a (poor resolution) bitmap version is available.
> 
> Proposed resolution:
> 
> * Reduce the number of logos, ideally to just one.
> * Generate an SVG version of this logo.
> * Update usage to match.
> * (Optionally) make a "media pack" containing the logo(s) in different formats available.
> * (Optionally) write guidelines on how to use the NeXus logo.
I think you were right as well.  It also will ease upkeep as well.  Although the range should still remain adaptable and seeded from, for, and exclusive to the individuated user of course.  The Name Will not be forgotten this way and we can actually ensure personal eAsset attribution as its likely the individual is not gonna want to change phones, or better platforms, when this all unfolds.
